lighthouse-core/gather/gatherers/dobetterweb/js-vulnerable-libraries.js

```diff
@@ -0,0 +1,96 @@
+
+
+  let libraries = {};
+  for (let i in lighthouseJSLibs) {
+    try {
+      const result = lighthouseJSLibs[i].test(window);
+      if (result === false) continue;
+      if (npmMapper[i].npmPkgName) {
+        libraries[npmMapper[i].npmPkgName] = {"version": result.version};
+      }
+    } catch(e) {
+      console.log('Library Detector test for ' + i + ' failed:', e);
```

Comment:
we wont' really be able to see these log statements easily, how about pushing these messages onto an array that we return along with `libraries`?